.env.example

```
@@ -176,4 +176,12 @@ AWS_BREEZE_PORTAL_URL="http://localhost:5173"
 GCP_BREEZE_PORTAL_URL="http://localhost:5174"
 
 DEFAULT_ANNOUNCEMENT_BANNER_BACKGROUND_COLOR=#714acd
-DEFAULT_ANNOUNCEMENT_BANNER_TEXT_COLOR=#714acd
\ No newline at end of file
+DEFAULT_ANNOUNCEMENT_BANNER_TEXT_COLOR=#714acd
+
+# Redis Configuration
+REDIS_HOST="localhost"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)